include/fuse_lowlevel.h

```diff
@@ -611,7 +611,9 @@ struct fuse_lowlevel_ops {
 	 * file: all file descriptors are closed and all memory mappings
 	 * are unmapped.
 	 *
-	 * For every open call there will be exactly one release call.
+	 * For every open call there will be exactly one release call,
+	 * excepting unmount, when all open file handles are implicitly
+	 * released.
```

Comment:
As discussed in the bug, let's research this more carefully. I believe even on unmount you should receive release() calls. If not, this is a bug that should be fixed in the code rather than documented.